src/openrct2/ride/RideRatings.cpp

```diff
@@ -2268,6 +2268,14 @@ bool RatingTuple::isNull() const
 void RatingTuple::setNull()
 {
     excitement = kRideRatingUndefined;
-    intensity = kRideRatingUndefined;
-    nausea = kRideRatingUndefined;
+    /*  reverting #22199 changes:
+    * 
+    *   It seems like excitement set to 0xFFFFu was intended
+    *   (guests going to untested rides is intended RCT behaviour
+    *   but setting intensity and nausea to 0xFFFFu to sort them
+    *   also made it so guests would get sick on untested rides
+    *   at a very high rate
+    */
+    // intensity = kRideRatingUndefined;
+    // nausea = kRideRatingUndefined;
```

Comment:
Remove all of that, for the history of the code we have git blame.

Reply:
roger - removed